docs/topics/readme/circuit-breaker-README.adoc

```diff
@@ -59,4 +59,4 @@ $ curl http://${greeting-app-name}-MY_PROJECT_NAME.LOCAL_OPENSHIFT_HOSTNAME/api/
 ----
 
 == CI/CD Deployment
-As part of the process of creating this booster, launch.openshift.io set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster Web Console. You can also learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+As part of the process of creating this booster, https://developers.redhat.com/launch set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster Web Console. You can also learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
```

Comment:
Just checking—is the URL hardcoded here because the README is shown as is, without rendering using Asciidoctor?

Reply:
@tradej correct. this file needs to be read from source as well as rendered adoc.